src/lib/__tests__/plugins.test.ts

```diff
@@ -423,7 +422,7 @@ describe('editor view', () => {
     expect(sectionWithList.childCount).toBe(2)
   })
 
-  test('loads with plugins', () => {
+  test.skip('loads with plugins', () => {
     const decoder = new Decoder(modelMap)
```

Comment:
this tests have to fixed

Reply:
this will be handled later in a cleanup ticket https://jira.atypon.com/browse/LEAN-3126

Reply:
fixed under LEAN-3030.